mantidproject/mantid
# TOFTOF GUI: improve save reduced data options

**Description of work.**

Refactored save reduced data block:
 - data formats can be chosen for `S(Q, W)` and `S(2theta, W)` independently
 - `ascii` option has been activated

**To test:**


Start the TOFTOF data reduction GUI: `Interfaces`&rarr;`Direct`&rarr;`DGS Reduction`. Choose facility MLZ and instrument TOFTOF.

Get the test data
[testdata.zip](https://github.com/mantidproject/mantid/files/1356265/testdata.zip)

and make sure, that those workspaces and file formats are saved, whos corresponding checkboxes are checked *and enabled*.

Fixes #22712. 

Does this update require release notes?



---
#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
